client/mm/libxc/binance.go

```diff
@@ -83,28 +81,74 @@ type bncAssetConfig struct {
 	conversionFactor uint64
 }
 
-func bncSymbolData(symbol string) (*bncAssetConfig, error) {
+// TODO: check all symbols
+var dexToBinanceSymbol = map[string]string{
+	"POLYGON": "MATIC",
+	"WETH":    "ETH",
```

Comment:
I don't understand the intent here. We can't deposit or withdraw weth.

Reply:
Yeah, it doesn't make a difference now. If we add `weth.arbitrum` then it will be used though.